nesta/production/orms/nih_es_config.json

```diff
@@ -86,31 +58,80 @@
                 "id_of_continent": {
                     "type": "keyword"
                 },
-                "terms_mesh_abstract": {
+                "id_of_project": {
+                  "type": "keyword"
+                },
+                "placeName_city_organisation": {
                     "type": "text",
                     "fields": {
                         "keyword": {
                             "type": "keyword"
                         }
-                    },
-                    "analyzer": "mesh_terms_analyzer"
+                    }
                 },
-                "booleanFlag_duplicate_abstract": {
-                    "type": "boolean",
-                    "null_value": false
+                "placeName_continent_organisation": {
```

Comment:
continents are single words, so I guess this is redundant

Reply:
The reason I added `text`here is that if I got it right with `keyword` looking for "asia" won't find ["Asia"](https://nesta-open-data.s3.eu-west-2.amazonaws.com/rwjf-viz/iso_a2_to_name_by_type.json), correct?

Reply:
good point!

Reply:
oh wait, no your logic is the wrong way around: `keyword` is case insensitive so "Asia" and "asia" will both return "Asia"

Reply:
Ah I see, good to know! Then I'd say `text` can go